ts/Series/BoxPlot/BoxPlotSeriesOptions.d.ts

```diff
@@ -14,31 +14,376 @@
  *
  * */
 
-import type BoxPlotPoint from './BoxPlotPoint';
-import type BoxPlotSeries from './BoxPlotSeries';
-import type ColumnSeriesOptions from '../Column/ColumnSeriesOptions';
-import type { SeriesStatesOptions } from '../../Core/Series/SeriesOptions';
+import BoxPlotPoint from './BoxPlotPoint';
+import BoxPlotPointOptions from './BoxPlotPointOptions';
+import BoxPlotSeries from './BoxPlotSeries';
+import ColumnSeriesOptions from '../Column/ColumnSeriesOptions';
+import { SeriesStatesOptions } from '../../Core/Series/SeriesOptions';
 
 /* *
  *
  *  Declarations
  *
  * */
 
+/**
+ * A box plot is a convenient way of depicting groups of data through their
+ * five-number summaries: the smallest observation (sample minimum), lower
+ * quartile (Q1), median (Q2), upper quartile (Q3), and largest observation
+ * (sample maximum).
+ *
+ * @sample highcharts/demo/box-plot/
+ *         Box plot
+ *
+ * @sample {highcharts} highcharts/css/boxplot/
+ *         Box plot in styled mode
+ *
+ * @sample {highcharts} highcharts/series-scatter/jitter-boxplot
+ *         Jittered scatter plot on top of a box plot
+ *
+ * @extends   plotOptions.column
+ * @excluding borderColor, borderRadius, borderWidth, groupZPadding,
+ *            states, boostThreshold, boostBlending
+ *
+ * @extends   series,plotOptions.boxplot
+ * @excluding dataParser, dataURL, marker, stack, stacking, states,
+ *            boostThreshold, boostBlending
+ *
+ * @product highcharts
+ *
+ * @requires highcharts-more
+ */
 export interface BoxPlotSeriesOptions extends ColumnSeriesOptions {
+
+    /**
+     * The dash style of the box.
+     *
+     * @sample {highcharts} highcharts/plotoptions/box-plot-styling/
+     *         Box plot styling
+     * @sample {highcharts} highcharts/css/boxplot/
+     *         Box plot in styled mode
+     *
+     * @type {Highcharts.DashStyleValue}
+     * @default Solid
+     * @since 8.1.0
+     * @product highcharts
+     * @apioption plotOptions.boxplot.boxDashStyle
```

Comment:
Can we now safely remove the `@apioption` tag? 🤔

Reply:
I have to discuss this with Markus. We might need these in the meantime until we can retire the JSDoc plugin.

Reply:
I will remove the apioption tags and Markus will reconstruct these for the JSDoc plugin. The type naming might need some hard-coded mapping, but in general it is detailed enough to build the option path. Only problem are camel case names in interfaces like `SeriesBoxPlotDataLabelOptions` etc.